test/parallel/test-crypto-ecdh-convert-key.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const crypto = require('crypto');
+
+const { ECDH, getCurves } = crypto;
```

Comment:
Tiny nit: This can be combined with the call to require:

```js
const assert = require('assert');
const { ECDH, getCurves } = require('crypto');
```